cmd/livepeer_cli/wizard_broadcast.go

```diff
@@ -51,8 +51,15 @@ func (w *wizard) allTranscodingOptions() map[int]string {
 }
 
 func (w *wizard) setBroadcastConfig() {
-	fmt.Printf("Enter broadcast max price per segment in Wei - ")
-	maxPricePerSegment := w.readBigInt()
+	fmt.Printf("Enter a broadcast max price per pixels transcoded in Wei.\n")
```

Comment:
Should we use "wei" instead of "Wei" since we use "wei" elsewhere (i.e. later in this prompt)?